cmd/osbuild-composer/composer.go

```diff
@@ -311,6 +311,24 @@ func (c *Composer) Start() error {
 				panic(err)
 			}
 		}()
+
+		// metrics listener on another port
+		metricsMux := http.NewServeMux()
+		metricsMux.Handle("/metrics", promhttp.Handler().(http.HandlerFunc))
+
+		addr := ":9090"
```

Comment:
Cockpit uses the same port number number, which is documented as "Web-based System Manager".

Moreover, shouldn't this be treated the same way as any other socket that we currently open? IOW use systemd socket unit and socket activation to get a new socket, instead of hardcoding the port and listening on all available interfaces by default...

Reply:
you're right, +1 on a different port.

So this was mostly intended for use in the container, but we might have other uses for the cloudapi in future. So let's do that. We also need to update the container entrypoint in that case though.

Reply:
BTW, I suggest to use `8008` port for prometheus socket, since it is listed as another HTTP-alt port in IANA port assignment list.